db/migrate/20221029002308_create_insurance_plans.rb

```diff
@@ -0,0 +1,13 @@
+# Migration for InsurancePlan
+class CreateInsurancePlans < ActiveRecord::Migration[7.0]
+  def change
+    create_table :insurance_plans do |t|
+      t.date :contract_date, null: false
+      t.references :car, null: false, foreign_key: true
+      t.integer :amount, null: false
+      t.date :exp_date
```

Comment:
agregar index a exp_date
usar nombre completo no abrevidado expiry_date creo que es
falta agregar references to proveedor